pkg/image/apis/image/v1/conversion.go

```diff
@@ -77,9 +77,7 @@ func Convert_image_Image_To_v1_Image(in *newer.Image, out *v1.Image, s conversio
 
 	if in.DockerImageSignatures != nil {
 		out.DockerImageSignatures = nil
```

Comment:
Now that you're at it, can you initialize `out.DockerImageSignature` I kinda feel this reads wrong. I know that I might have written it can you change that into:
 ```
out.DockerImageSignatures = make([][]byte, 0, len(in.DockerImageSignatures)
```
This way we'll save some cycles and conversions are always expensive, so it's good if we can do so.

Reply:
addressed